GoogleForCreators/web-stories-wp
# Fix Play button style issues

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
| Before | After |
|-------|-------|
| ![Screenshot 2021-05-21 at 13 08 39](https://user-images.githubusercontent.com/3294597/119121088-4f337880-ba2d-11eb-9aaf-738e886a9eba.png) | ![Screenshot 2021-05-21 at 13 05 35](https://user-images.githubusercontent.com/3294597/119120918-1abfbc80-ba2d-11eb-8907-86291b556d67.png) |
| ![Screenshot 2021-05-21 at 13 08 16](https://user-images.githubusercontent.com/3294597/119121130-59557700-ba2d-11eb-8200-992c4b9cda15.png) | ![Screenshot 2021-05-21 at 13 07 05](https://user-images.githubusercontent.com/3294597/119120946-23b08e00-ba2d-11eb-83dd-b6eaa8ae6e54.png) |


## Testing Instructions
I
1. Set your computer to always show scrollbars
2. Add a video element
3. Make the video element narrow enough for the play button to show above it instead of on top of it.
4. Verify the play button does not show scrollbars.

II
1. Switch the editor to RTL
2. Add a video element
3. Select the video element and hover to see the play button, verify it displays centred in the middle of the video.



### QA
See above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7397
